### PR TITLE
Update THIRDPARTY

### PR DESCRIPTION
**Pull request by oliveralka**
Update Thirdparty 
Fix omssa macOS 10.15 issue (https://github.com/OpenMS/OpenMS/issues/4281)
Reactivate third-party test (omssa macOS)

### CODE REVIEW

File: src/tests/topp/THIRDPARTY/third_party_tests.cmake [hunk 1]
```diff
@@ -104,7 +104,6 @@ OPENMS_FINDBINARY(LUCIPHOR_BINARY "luciphor2.jar" "LuciPHOr2")
 #------------------------------------------------------------------------------
 ## optional tests
 if (NOT (${OMSSA_BINARY} STREQUAL "OMSSA_BINARY-NOTFOUND"))
-  if (NOT APPLE OR (CMAKE_SYSTEM_VERSION VERSION_LESS 10.15)) ## macOS Catalina does not support 32-bit apps anymore.
```
Comment (jpfeuffer) [inline src/tests/topp/THIRDPARTY/third_party_tests.cmake:107]: forgot to remove endif

Comment (oliveralka) [reply src/tests/topp/THIRDPARTY/third_party_tests.cmake:107]: damn :D

**Comment by oliveralka** [comment]
```
dyld: lazy symbol binding failed: Symbol not found: ____chkstk_darwin
  Referenced from: /Users/builder/jenkins/ws/openms/PR/bldtst/c797036f/SEARCHENGINES/OMSSA/omssacl
  Expected in: flat namespace

dyld: Symbol not found: ____chkstk_darwin
  Referenced from: /Users/builder/jenkins/ws/openms/PR/bldtst/c797036f/SEARCHENGINES/OMSSA/omssacl
  Expected in: flat namespace
```

I guess wrong deployment target - minos 10.15.
Will try to fix it.

**Comment by oliveralka** [comment]
@jpfeuffer I am not sure what is going on - Jenkins issue.

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Pull request merged**